test/service/test_response.py

```diff
@@ -1500,12 +1501,22 @@ def test_integrations_by_entity_ids(self, portal_integrations_db):
              ['f13ddf2d-d913-492b-9ea8-2de4b1881c26', 'dbfe9394-a326-4574-9632-fbadb51a7b1a']),
             # One project entity id specified by two different integrations
             (['c4077b3c-5c98-4d26-a614-246d12c2e5d7'],
-             ['977854a0-2eea-4fec-9459-d4807fe79f0c', 'e8b3ca4f-bcf5-42eb-b58c-de6d7e0fe138'])
+             ['977854a0-2eea-4fec-9459-d4807fe79f0c', 'e8b3ca4f-bcf5-42eb-b58c-de6d7e0fe138']),
+            # Blank entity id, to match integrations lacking the entity_id field (in this case none of them)
```

Comment:
Please add such an integration for this case to be more meaningful.

Reply:
Add it to the mock database that is.

Reply:
The current implementation (in my branch and in [develop](https://github.com/DataBiosphere/azul/blob/develop/lambdas/service/app.py#L421)) is to include integrations lacking the `entity_ids` field even when a non-null `entity_ids` query parameter is provided. I noticed this when the unit tests failed due to the new entry (missing the field) being expected in the results of every other subtest. I just wanted to confirm whether this is still the desired behavior, i.e., the new entry should be added to the other tests.